### PR TITLE
Update Travis configuration matrix and README to reflect changes made to support specific TF versions and deprecate Theano support

### PR DESCRIPTION
**Pull request by npapernot**

### CODE REVIEW

**Comment by carlini** [comment]
Should this wait until bug 150 is decided and resolved?

**Comment by goodfeli** [comment]
Yeah, let's resolve #150 before we increment the tag.

**Comment by npapernot** [comment]
We should also close #145 at the same time

File: .travis.yml [hunk 1]
```diff
@@ -5,9 +5,21 @@ language: python
 matrix:
     include:
         - python: 2.7
-          env: KERAS_BACKEND=tensorflow
+          env:
+            - KERAS_BACKEND=tensorflow
+            - TENSORFLOW_V=1.0.1
```
Comment (goodfeli) [inline .travis.yml]: We should also test tf versions 1.1 and 1.2, right? We don't need to get all combinations of python version and tensorflow version, but I think we should get all supported minor versions of tensorflow.

Comment (npapernot) [inline .travis.yml]: I thought we were going for the last stable release (at the moment 1.1), and all past major versions (only 1.0 at the moment) ? This is what I meant here: https://groups.google.com/forum/#!topic/cleverhans-dev/TlyFojSkepU

Did you have something else in mind like all past major versions and all minor versions after the last major?

Comment (goodfeli) [inline .travis.yml]: I think this is OK. I had made two mistakes:
- I didn't see that we do test v1.1 already; it just doesn't show up in the diff for this PR.
- I saw that a TensorFlow v1.2 is available, but it is not considered stable yet so we don't need to support it yet

File: README.md [hunk 2]
```diff
@@ -68,26 +64,28 @@ export PYTHONPATH="/path/to/cleverhans":$PYTHONPATH
 
 You may want to make that change permanent through your shell's profile.
 
+### Currently supported setups
+
+Although CleverHans is likely to work on many other machine configurations, we
+currently [test it](https://travis-ci.org/tensorflow/cleverhans) with Python
+{2.7, 3.5} and TensorFlow {1.0, 1.2} on Ubuntu 14.04.5 LTS (Trusty Tahr).
```
Comment (goodfeli) [inline README.md]: Typo, 1.2 instead of 1.1

File: README.md [hunk 3]
```diff
@@ -97,35 +95,44 @@ carefully by the adversary. The adversary then uses the substitute
...
 When reporting benchmarks, please:
-* Use a versioned release of `CleverHans`. You can find a list of released versions [here](https://github.com/tensorflow/cleverhans/releases).
+* Use a versioned release of CleverHans. You can find a list of released versions [here](https://github.com/tensorflow/cleverhans/releases).
 * Either use the latest version, or, if comparing to an earlier publication, use the same version as the earlier publication.
 * Report which attack method was used.
 * Report any configuration variables used to determine the behavior of the attack.
 
 For example, you might report "We benchmarked the robustness of our method to
-adversarial attack using v1.0.0 of `CleverHans`. On a test set modified by the
+adversarial attack using v2.0.0 of CleverHans. On a test set modified by the
```
Comment (goodfeli) [inline README.md]: This part still increments version number

File: README.md (hunk 2 above)
Comment (npapernot) [inline README.md]: Fixed

File: README.md (hunk 3 above)
Comment (npapernot) [inline README.md]: Fixed

**Pull request merged**